radio/src/targets/horus/board.h

```diff
@@ -153,10 +153,18 @@ void SDRAM_Init();
   #define INTERNAL_MODULE_OFF()        GPIO_ResetBits(INTMODULE_PWR_GPIO, INTMODULE_PWR_GPIO_PIN)
 #endif
 
-#define EXTERNAL_MODULE_ON()           GPIO_SetBits(EXTMODULE_PWR_GPIO, EXTMODULE_PWR_GPIO_PIN)
-#define EXTERNAL_MODULE_OFF()          GPIO_ResetBits(EXTMODULE_PWR_GPIO, EXTMODULE_PWR_GPIO_PIN)
+//OW
+//#define EXTERNAL_MODULE_ON()           GPIO_SetBits(EXTMODULE_PWR_GPIO, EXTMODULE_PWR_GPIO_PIN)
```

Comment:
Interesting: these defs are not used normally?

Reply:
they are used, that's why I repurpose them

Reply:
Sorry, I think I don't understand the "re-purposing" :-( Could you please elaborate on that?

Reply:
well, these macros are used throughout the code, but in their original form they don't do what I need them to do ... so I either could have reworked all the code or simply replace them and make them call what is needed ... quite standard approach and in fact used lots and lots in the opentx/edgetx code ...